src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj

```diff
@@ -15,16 +15,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.4" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
     <PackageReference Include="System.Xml.XPath" Version="4.0.1" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.3" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
```

Comment:
This is already included on line 20

Reply:
Isn't line 20 for netstandard2.0 and this one for netstandard1.4? Before my update also, they had this package at 2 places.

Reply:
No, line `20` and `21` has no condition on the `ItemGroup`, so it applies to all TFMs.

I imagine this was a mistake before your change to have the package in both places